scripts/markdown-magic.config.js

```diff
@@ -20,11 +20,11 @@ exports.transforms = {
   usage: (content, options) => {
     const {executable} = options;
     const flag = options.flag || '--help';
-    const header = options.header || '\n```plain';
+    const header = options.header || '\n```text';
```

Comment:
does this actually matter or make a difference? curious

Reply:
Not sure what it does if language is incorrect. Looked around earlier for what language was in use if none specified, but couldn't determine that either.

"text" seemed closest to your intent based on [languages](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) documentation.